en/hiring_blockchain_research_mentor.md

```diff
@@ -0,0 +1,78 @@
+
+SCRF is seeking a highly motivated individual who has insight into the state-of-the-art blockchain technology advancement, an interest in research discussions, and a passion for mentoring blockchain research enthusiasts.
+
+
+## SCRF Mission
+
+SCRF is a premier, open discourse hub dedicated to web3 advancement with tools in place to amplify the latest quality research, reward contributions, deliver insights, and facilitate industry connections. Composed of an active international community, SCRF connects academic researchers with blockchain innovators to collaborate on solutions.
+
+
+## Role Summary
+
+The Blockchain Research Mentor will use their expertise in blockchain to guide research content creation on the forum. By working closely with individual SCRF’s community members, the mentor will propose, review, and guide mentees to write about focused topics on the forum. The Blockchain Research Mentor helps curate research content to be accurate and insightful.
+
+
+## Responsibilities
+
+The Blockchain Research Mentor is expected to:
+
+
+
+* Propose ideas for web3 research content that aligns with SCRF’s mission
+* Review and improve research content ideas generated by the community
+* Help define the scope of content to ensure it fits a clear timeline from start to completion
+* Provide constructive feedback and guidance for contributors creating ongoing research content
+* Offer insight around the importance and relevancy of a piece of research to the challenges faced in the blockchain/web3 space
+
+
+## Qualifications
+
+SCRF is looking for someone who has knowledge of technical discipline related to blockchain, web3, or cryptocurrencies.
+
+Qualified candidates have:
+
+
+
+
+* An academic background (preferably) or equivalent years of work experience
+* A general understanding of blockchain-related research spaces is necessary for this role or, ideally, a specialization in a sub-field of blockchain research
+* Experience or strong interest in science communication
+* Clear communication skills and willingness to use chat, video, and email to coordinate with team members and mentees
+* Ability to work with a high-paced, remote, and globally distributed team
+
+
+## Compensation and Logistics
+
+Work at SCRF includes:
+
+
+
+* Fully remote
+* Negotiable, competitive pay at $30-$50/hour to start 
+* Part-time (10-15 hours) to start, opportunities to increase with time 
+* Compensation in cryptocurrency preferred
+
+
+## How to Apply
+
+To apply for this role, please join SCRF’s Discord server and follow the instructions below. Further application instructions are available on [💲start-here](https://discord.com/channels/784234332617048065/962841663246585896) and [🚅start-here Discord channels.](https://discord.com/channels/784234332617048065/968215614550323210)
```

Comment:
please refer to our platforms as the function they perform. i.e. we have a chat, not a Discord and a forum, not a Discourse.